client/components/plans/plan-price/index.jsx

```diff
@@ -62,13 +62,30 @@ module.exports = React.createClass( {
 			periodLabel = hasDiscount ? this.translate( 'due today when you upgrade' ) : plan.bill_period_label
 		}
 
-		return (
-			<div className={ hasDiscount ? "plan-price plan-price__discount" : "plan-price" }>
-				<span>{ this.getPrice() }</span>
-				<small className="plan-price__billing-period">
-					{ periodLabel }
-				</small>
-			</div>
-		);
+		if ( site && site.jetpack ) {
+			return (
+				<div className="plan-price">
+					<span className="jetpack-price">{ plan.formatted_original_price }</span>
+					<small className="plan-price__billing-period">
+						{ this.translate( 'cost of individual plugins' ) }
+		      </small>
+		      <span className="jetpack-price">{ this.getPrice() }</span>
+		      <small className="plan-price__billing-period">
+		        { hasDiscount ? this.translate( 'for first year' ) : plan.bill_period_label } (
+		        { plan.saving }
+		        % { this.translate( 'savings' ) })
+		      </small>
+		    </div>
+		  );
+		} else {
+			return (
+				<div className={ hasDiscount ? "plan-price plan-price__discount" : "plan-price" }>
```

Comment:
Do you mean on line #82?

I didn't change that and I assumed it is like that to match line #67 ?